default.nix

```diff
@@ -1,3 +1,4 @@
 (import (fetchTarball https://github.com/edolstra/flake-compat/archive/master.tar.gz) {
   src = builtins.fetchGit ./.;
-}).defaultNix
+}
+).defaultNix
```

Comment:
I run ``nixfmt`` on your branch and got quite a bit of changes. Not sure they should be incorporated. Your call.